rosplan_demo_waypoints:
    waypoint_frameid: map
    home: [0.22996230442059445, 4.847318911974817, 0.0005454411783606949]
    wp10: [0.0, 2.0, -0.0]
    wp1: [48.0, 48.0, -0.0]
    wp2: [50.0, 50.0, 0.0]
rosplan_waypoints_goal_tolerance:
    home: [0.01, 3.62]
    wp10: [0.01, 3.62]
    wp1: [0.01, 3.62]
    wp2: [0.01, 3.62]
